builtin/lookup.go

```diff
@@ -769,12 +763,6 @@ option::local includePatterns(variadic string pattern)
 # @return an option to sync files that don&#39;t match any pattern.
 option::local excludePatterns(variadic string pattern)
 
-# Sync the targets of symlinks if path is to a symlink.
-#
-# @param path a list of paths to files that may be symlinks.
-# @return an option to sync the targets of symlinks.
-option::local followPaths(variadic string path)
```

Comment:
`llb.FollowPaths` unfortunately doesn't behave the same as copy's `(llb.CopyInfo).FollowSymlinks`. Very similar, but not identical.

If upstream can converge these two, we can support this again.

Reply:
I'm curious what the difference is.

Reply:
For `llb.FollowPaths`, it uses `FollowLinks` from here:
https://github.com/tonistiigi/fsutil/blob/master/walker.go#L49

For `(llb.CopyInfo).FollowSymlinks`, it uses `rootPath` from here (`ResolveWildcards` ends up using `rootPath` too):
https://github.com/tonistiigi/fsutil/blob/master/copy/copy.go#L110

which ends up using `RootPath` from `containerd/continuity`:
https://github.com/containerd/continuity/blob/main/fs/path.go#L225-L227